### ISSUE TITLE
Calendar A11y Error: Buttons must have discernable text.

### ISSUE THREAD

**Issue by CharlesgdHill** [OfficeDev/office-ui-fabric-react#8285]
Calendar month navigations fail accessibility fastpass.

Please provide a reproduction of the bug in a codepen:
https://fabricweb.z5.web.core.windows.net/oufr/6.155.0/#/examples/calendar

#### Actual behavior:
 

![image](https://user-images.githubusercontent.com/44102734/54236550-0688e800-44d1-11e9-87d0-43acb02807f6.png)

 
![image](https://user-images.githubusercontent.com/44102734/54236600-24564d00-44d1-11e9-8ee1-dd933d5a69ac.png)

#### Expected behavior:
 
No accessibility error

Requested priority: Blocking. We must have no issues from accessibility FastPass.

**Comment by KevinTCoughlin** [comment]
@OfficeDev/uifabric-calendar @lorejoh12 are you folks tracking contrast issues for the current Calendar control? It looks like some folks are blocked by it, please see above.

**Comment by KevinTCoughlin** [comment]
I spoke with @lorejoh12 today who recommended we put Calendar related issues under the date-time project so that it receives proper attention by the feature crew.

**Comment by lorejoh12** [comment]
This issue was flagged by fast-pass on Fabric v6, and no longer repros with the latest version of OUFR 7.59.3. Closing.
